app.js

```diff
@@ -1,8 +1,23 @@
 const express = require("express");
-const getApi = require("./controller/api.controller");
+const { getApi, getTopics } = require("./controller/api.controller");
 
 const app = express();
+app.use(express.json());
```

Comment:
This is only relevant for post or patch requests. This code is not needed for a get and outside the scope of the ticket.